index.html

```diff
@@ -4,9 +4,69 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" href="style.css">
+    <script src="https://kit.fontawesome.com/fdedc31e53.js" crossorigin="anonymous"></script>
+    <link href="https://fonts.googleapis.com/css?family=Nunito&display=swap" rel="stylesheet">
     <title>Apple.com</title>
 </head>
 <body>
+    <div class="container">
+        <header>
+            <nav>
+                <ul class="globalnav">
+                    <li class="list" id="gn-apple"></li>
+                    <li class="list" id="gn-store"></li>
+                    <li class="list" id="gn-mac"></li>
+                    <li class="list" id="gn-ipod"></li>
+                    <li class="list" id="gn-iphone"></li>
+                    <li class="list" id="gn-ipad"></li>
+                    <li class="list" id="gn-itunes"></li>
+                    <li class="list" id="gn-support"></li>
+                    <li id="search">
+                        <form>
+                            <i class="fas fa-search fa-sm"></i>
+                            <input type="text" name="search">
+                        </form>
+                    </li>
+                </ul>
+            </nav>
+            <section class="header-text">
+                <p class="top">What will your verse be</p>
+                <p class="middle">Everyone has something to share. See how people are using iPad in their unique ways to add to the world's story</p>
+                <p class="bottom">Watch the "Your Verse" film and explore the stories </p>
+            </section>
+        </header>
+    
+    </div>
+    <div class="stuff">
+        <img src="./img/5s.jpg" alt="">
+        <img src="./img/5c.jpg" alt="">
+        <img src="./img/30yrs.jpg" alt="">
+        <img src="./img/supplier.jpg" alt="">
+    </div>
```

Comment:
Each of these `img` tags should be wrapped in an `a` tag since they are links.

#### Bottom

* [ ] It contains four elements which are links (to another articles or `#`) ([link to page](https://web.archive.org/web/20140301004610/http://www.apple.com/))